test/bin/build_images.sh

```diff
@@ -25,6 +25,29 @@ osbuild_logs() {
     done
 }
 
+extract_container_images() {
+    local -r version=$1
+    local -r repo_url=$2
+    local -r outfile=$3
+
+    mkdir -p "${IMAGEDIR}/release-info-rpms"
+    pushd "${IMAGEDIR}/release-info-rpms"
+    sudo bash -c 'cat > /etc/yum.repos.d/microshift.repo' << EOF
```

Comment:
Can we use a temporary name to avoid collisions?

Reply:
You can not have collisions in the way the function is called. Only one rpm at a time, with its own repo. At the end, remove everything, and if something goes wrong the script stops. On the next run (picture local development) it will be overwritten.
Or did I get you wrong?